scalameta/scalameta/src/test/scala/scala/meta/tests/prettyprinters/SyntacticSuite.scala

```diff
@@ -7,14 +7,14 @@ import scala.meta.dialects.Scala211
 import scala.meta.internal.ast._
 
 class SyntacticSuite extends scala.meta.tests.parsers.ParseSuite {
-  override def term(code: String)(implicit dialect: Dialect) = super.term(code).resetAllOrigins
-  override def pat(code: String)(implicit dialect: Dialect) = super.pat(code).resetAllOrigins
-  override def tpe(code: String)(implicit dialect: Dialect) = super.tpe(code).resetAllOrigins
-  override def topStat(code: String)(implicit dialect: Dialect) = super.topStat(code).resetAllOrigins
-  override def templStat(code: String)(implicit dialect: Dialect) = super.templStat(code).resetAllOrigins
-  override def blockStat(code: String)(implicit dialect: Dialect) = super.blockStat(code).resetAllOrigins
-  override def caseClause(code: String)(implicit dialect: Dialect) = super.caseClause(code).resetAllOrigins
-  override def source(code: String)(implicit dialect: Dialect) = super.source(code).resetAllOrigins
+  override def term(code: String)(implicit dialect: Dialect)       = super.term(code)(dialect).resetAllOrigins
```

Comment:
Yes, I was surprised by this - the `import scala.meta.dialects.Scala211` at the top of the file takes precedence over the implicit `dialect` parameter to the function when calling `super.tpe(...)`